src/hooks/useSwapCurrencyHandlers.js

```diff
@@ -131,7 +136,7 @@ export default function useSwapCurrencyHandlers({
       const newInputCurrency = inputCurrency
         ? {
             ...inputCurrency,
-            type: ethereumUtils.getNetworkFromType(inputCurrency?.type),
```

Comment:
`getNetworkFromType` this was broke right? we were using network type, curious about that line